### ISSUE TITLE
UIDesigner is not building due to missing artifact

### ISSUE THREAD

**Issue by laurentleseigneur** [Bonitasoft-Community/Build-Bonita#19]
`ui-designer-backend-webapp` step is failing because `org.bonitasoft.web:ui-designer-backend-contract:jar:tests:1.9.53`is not build before (may be due to "skip test")
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] ui-designer ........................................ SUCCESS [  0.218 s]
[INFO] ui-designer-frontend ............................... SUCCESS [01:27 min]
[INFO] ui-designer-backend ................................ SUCCESS [ 11.872 s]
[INFO] ui-designer-backend-contract ....................... SUCCESS [  8.951 s]
[INFO] ui-designer-backend-webapp ......................... FAILURE [ 13.304 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:01 min
[INFO] Finished at: 2019-09-26T11:36:22+02:00
[INFO] Final Memory: 28M/369M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project ui-designer-backend-webapp: Could not resolve dependencies for project org.bonitasoft.web:ui-designer-backend-webapp:war:1.9.53: Could not find artifact org.bonitasoft.web:ui-designer-backend-contract:jar:tests:1.9.53 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR]
```

**Comment by amottier** [comment]
The issue is caused by `maven.test.skip` and related to https://issues.apache.org/jira/browse/MJAR-138
In my attempt to build 7.9.3 I face this issue and modify the script locally to only to `-DskipTests`.
I face other issues related to 7.9.3 build that prevent me from pushing this version. As 7.9.4 is now available on should fix the issue reported on 7.9.3 I will try again.

**Comment by amottier** [comment]
Commit b8f9b5e fix this issue. Build script is fully functionnal on 7.9.4 with two remaining workarounds: one for Studio p2 repository (will be fixed in 7.10) and one for UI Designer (will be fixed in 7.9.5).
